lib/phoenix/endpoint/cowboy2_handler.ex

```diff
@@ -60,6 +60,7 @@ defmodule Phoenix.Endpoint.Cowboy2Handler do
 
         {:plug, conn, handler, opts} ->
           %{adapter: {@connection, req}} =
+            conn =
```

Comment:
Should we indent the subsequent lines?

```
            conn =
              conn
              |> handler.call(opts)
              |> maybe_send(handler)
```

Reply:
I'll format it however you want, I have the code formatter on in my editor and this is what it does...